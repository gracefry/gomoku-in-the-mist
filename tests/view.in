view
place A1
view
place B2
view
place C3
view
place D4
view
history
term