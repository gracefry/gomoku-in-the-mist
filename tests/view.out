J10,.................................................
M3,..........................................xxxxxxx
L17,xxxxxxx..........................................
B1,xx.....xx..#..xx.o...xx#....xxxxxxxxxxxxxxxxxxxxx
B12,xx.....xx.....xx.....xx.....xx.....xx.....xx.....
A1B2C3D4
